cms/jinja2/assets/js/hide-date-text-on-non-provisional-releases.js

```diff
@@ -0,0 +1,16 @@
+function toggleDateTextVisibility() {
+  var statusField = document.getElementById('id_status');
```

Comment:
Additionally what you could consider is using `event.target` which will be the element which triggered the change event, so you wouldn't need to re-retrieve it using `getElementById`.

https://developer.mozilla.org/en-US/docs/Web/API/Event/target

Reply:
I don't think that would work - note that `toggleDateTextVisibility` is called on DOM load even when there's no change to `status` to trigger it. This is to handle the case when the user opens editor on an already existing page that is non-provisional.

Reply:
That's a good point @kacperpONS – I'll resolve.